src/main/java/org/testng/internal/GroupConfigMethodArguments.java

```diff
@@ -3,18 +3,21 @@
 import java.util.Map;
 import org.testng.ITestNGMethod;
 import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
 
 public class GroupConfigMethodArguments extends Arguments {
 
   private final ConfigurationGroupMethods groupMethods;
   private final XmlSuite suite;
```

Comment:
With the introduction of XmlTest, suite could be removed with getSuite() -> return xmlTest.getSuite()

Reply:
fixed. I went ahead and refactored the need to feed in a suite xml object and instead had all of it retrieved from `ITestNGMethod` object